src/components/reusable/Graph.js

```diff
@@ -61,48 +67,41 @@ const buildGraph = (data, categories) => {
           style: {
             colors: "#1F302B",
           },
+          formatter: function (value) {
+            return value.toFixed(2); // Set the label value to have a maximum of 2 decimal points
+          },
         },
       },
     },
   };
 };
 
-const Graph = ({ getData, chartData, categories }) => {
-  // Create the graph with the data and categories along with the callback to get more data
-  let chart = buildGraph(chartData, categories);
-  const [selectedFilter, setSelectedFilter] = useState("Today");
-
-  const filters = {
-    DAY: "Today",
-    FIVE: "Past 5 days",
-    MONTH: "Past month",
-    YEAR: "Past Year",
-  };
+const Graph = ({ stockData, getData, xValues, yValues, selectedFilter }) => {
+  // conditional variable to indicate whether stock searched is trending up or down
+  const isTrendingUp = stockData[0].priceChange >= 0;
 
-  const handleFilterSelection = (filter) => {
-    getData(filter);
-    setSelectedFilter(filter);
-    // logic to update chart data based on selected filter
-  };
+  // Create the graph with the data and categories along with the callback to get more data
+  let chart = buildGraph(xValues, yValues, isTrendingUp);
 
   return (
     <div className="relative h-96">
-      <p className="inline text-green font-light"> {selectedFilter}</p>
       <Chart
         options={chart.options}
         series={chart.series}
         type="line"
         width="100%"
         height="100%"
+        className="shadow-md"
       />
+      {/* Tabs for users to interact with and fetch different data based on provided timeframes */}
       <div className="flex mt-4 justify-center">
         <button
           className={`py-2 px-4 text-green font-semibold hover:bg-smokewhite ${
             selectedFilter === filters.DAY
               ? "text-cokewhite border-b-green bg-green active hover:bg-green"
```

Comment:
This string should probably be a variable:

` UPTREND_SCREE = "text-cokewhite border-b-green bg-green active hover:bg-green"

Reply:
fixed